tests/base/test_pytorch_transforms.py

```diff
@@ -40,3 +138,387 @@ def forward(self, x):
     assert transformed
     y2 = model(x)
     assert torch.all(y2 == x)
+
+
+@pytest.mark.parametrize("input_size", [random.randint(0, 10)], ids=lambda x: "input_size=" + str(x))
+@pytest.mark.parametrize("hidden_size", random.sample([4, 64, 2048, 4096], 2), ids=lambda x: "hidden_size=" + str(x))
```

Comment:
We should not be picking hidden_size at random for each run. We should have a fixed set of parameters. Because when we have a test fail once, we'll not be reproduce when we run the next time.

And I think we should not be adding a common test for all custom ops like this. We should be adding tests for each custom op separately.